### PR TITLE
[CU-206ydj2] test(Runtime Integration Tests): Dep Update, Docker Integration, args.ts removal, mintingHelper added

### PR DESCRIPTION
**Pull request by Dom-Roth**
Greetings, here are a lot of minor things for the integration tests.

## Integration Test Update

- Added a docker-compose file, to set up a development chain and automatically run the test suite on it.
- Updated Dependencies
- Removed args.ts! New way to define endpoint is by setting the `ENDPOINT` environment variable.
- Added mintingHelper.ts to provide easy utility functions to mint assets for wallets in tests.

### Minor
- Disabled parallel mode due to the current issue with `priority too low` events. Will be re-enabled when this is fixed.
- Updated Types
- Removed ToDo part of README.md
- Some minor cleanup and refactoring.

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-206ydj2 Update Runtime Integration Test Project Dependencies](https://app.clickup.com/t/206ydj2)

**Review by Juliansv — APPROVED** [review]
Looks good to me.

**Review by kollegian — DISMISSED** [review]

File: integration-tests/runtime-tests/src/utils/testSetup.ts [hunk 1]
```diff
@@ -6,25 +6,30 @@ import * as definitions from '@composable/../../src/types/interfaces/definitions
...
+// Enable and inject BN dependency
+  chai.use(chai_bn(BN));
   const rpc = Object.keys(definitions).reduce((accumulator, key) => ({ ...accumulator, [key]: definitions[key].rpc }), {});
   const types = Object.values(definitions).reduce((accumulator, { types }) => ({ ...accumulator, ...types }), {});
 
-  global.endpoint = `ws://${args.h}:${args.p}`;
+  if (isEndpointDefined())
+    global.endpoint = `ws://${process.env.ENDPOINT}`;
+  else
+    global.endpoint = 'ws://127.0.0.1:9988';
```
Comment (andor0) [inline integration-tests/runtime-tests/src/utils/testSetup.ts:32]: maybe that's better
```
global.endpoint = 'ws://' + (process.env.ENDPOINT ?? '127.0.0.1:9988');
```
?

**Comment by andor0** [comment]
can add how to run docker-compose in README.md

**Review by andor0 — DISMISSED** [review]

**Review by andor0 — APPROVED** [review]

File: integration-tests/runtime-tests/src/utils/testSetup.ts (hunk 1 above)
Comment (Dom-Roth) [reply integration-tests/runtime-tests/src/utils/testSetup.ts]: Way better! Thanks a lot. :+1:

**Review by hussein-aitlahcen — APPROVED** [review]

**Pull request merged**